apps/functions/applications-migration/common/migrators/nsip-representation-migration.js

```diff
@@ -46,19 +46,29 @@ export const getRepresentationsForCase = async (log, caseReference) => {
 			...row,
 			representationId: parseInt(row.representationId),
 			caseId: parseInt(row.caseId),
-			originalRepresentation: constructOriginalRepresentation(
-				row.originalRepresentation,
-				row.redactedRepresentation
-			),
+			...handleRepresentationText(row.originalRepresentation, row.redactedRepresentation),
 			attachmentIds: valueToArray(row.attachmentIds)
 		};
 	});
 
 	return { representationEntities, count };
 };
 
-const constructOriginalRepresentation = (originalRepresentation, redactedRepresentation) => {
-	if (originalRepresentation?.trim()) return originalRepresentation.trim();
-	if (redactedRepresentation?.trim()) return redactedRepresentation.trim();
-	return 'No data in original record';
+/**
+ *
+ * @param {string | null} originalRepresentation
+ * @param {string | null} redactedRepresentation
+ */
+const handleRepresentationText = (originalRepresentation, redactedRepresentation) => {
+	const fallbackText = 'No data in original  Horizon record';
+	if (!originalRepresentation && !redactedRepresentation) {
+		return { originalRepresentation: null, redactedRepresentation: null };
+	}
+	if (!originalRepresentation && redactedRepresentation) {
+		return { originalRepresentation: fallbackText, redactedRepresentation };
+	}
+	if (originalRepresentation && !redactedRepresentation) {
+		return { originalRepresentation, redactedRepresentation: null };
+	}
```

Comment:
Based on how the data is in ODW currently, yes. It's more just to cover all bases really

Reply:
Happy to leave it if you want. Although its redundant and the same thing will happen by the bottom return, this is a bit more explicit and confirms the desired behaviour. Only other thing to ask was, why dont we use if / else if./ else instead of only ifs? Again not a big thing that would be more standard.